src/main/java/com/github/checkstyle/regression/data/GitChange.java

```diff
@@ -32,4 +34,18 @@
      * @return the path of the changed file
      */
     String path();
+
+    /**
+     * The line numbers of the added codes.
```

Comment:
What is `code` here? We can't be sure new lines are code, comments, etc...
Should this just be changes?